app/assets/stylesheets/student_index.scss

```diff
@@ -0,0 +1,54 @@
+@import 'color';
+
+.student_index_container {
+  margin-top: 40px;
+
+  .student_block {
+    margin-right: 10px;
```

Comment:
Properties should be ordered background-color, border-color, border-style, border-width, height, margin-bottom, margin-right